documentation/docs/40-best-practices/05-performance.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Performance
+---
+
+SvelteKit strives to provide a highly performant experience by default. However, when building any web application you will always need to put in some effort to maintain fast speeds. Some tips are included below.
```

Comment:
Converting @FractalHQ's edits into individual suggestions

```suggestion
SvelteKit apps provide highly performant experiences by default, but it's always good to invest time to ensure your code is as performant as it can be. Here are some tips to help you get the most out of your SvelteKit app.
```